### ISSUE TITLE
create function to display filtered books list

### ISSUE THREAD

**Issue by bdolor** [BCcampus/opentextbooks#132]
**Is your feature request related to a problem? Please describe.**
Re-design requires a new display of books, based on a curated list. Blocked by https://github.com/BCcampus/opentextbooks/issues/130

**Describe the solution you'd like**
An HTML blob of filtered books, each with  
- Links to cover images
- Title of book 
- Date published

**Describe alternatives you've considered**
can't go under it, go over it, can't go around it...we have to go through it. This is the only way.

**Comment by bdolor** [comment]
duplicate of https://github.com/BCcampus/opentextbooks/issues/123
